src/Offer/CommandHandlers/UpdateContributorsHandler.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Offer\CommandHandlers;
+
+use Broadway\CommandHandling\CommandHandler;
+use CultuurNet\UDB3\Contributor\ContributorRepository;
+use CultuurNet\UDB3\Model\ValueObject\Identity\UUID;
+use CultuurNet\UDB3\Offer\Commands\UpdateContributors;
+use CultuurNet\UDB3\Offer\OfferRepository;
+
+final class UpdateContributorsHandler implements CommandHandler
+{
+    private OfferRepository $offerRepository;
+
+    private ContributorRepository $contributorRepository;
+
+    public function __construct(OfferRepository $offerRepository, ContributorRepository $contributorRepository)
+    {
+        $this->offerRepository = $offerRepository;
+        $this->contributorRepository = $contributorRepository;
+    }
+    public function handle($command): void
+    {
+        if (!($command instanceof UpdateContributors)) {
+            return;
+        }
+
+        $this->offerRepository->load($command->getItemId());
```

Comment:
Nit: Maybe add a comment here that you load the offer to check that it actually exists